core/src/main/java/bisq/core/dao/node/parser/TxOutputParser.java

```diff
@@ -118,19 +170,39 @@ void processTxOutput(TempTxOutput tempTxOutput) {
             } else if (isBtcOutputOfBurnFeeTx(tempTxOutput)) {
                 // In case we have the opReturn for a burn fee tx all outputs after 1st output are considered BTC
                 handleBtcOutput(tempTxOutput, index);
+            } else if (isHardForkActivated(tempTxOutput) && isIssuanceCandidateTxOutput(tempTxOutput)) {
+                // After the hard fork activation we fix a bug with a transaction which would have interpreted the
+                // vote fee output as BSQ if the vote fee was >= miner fee.
+                // Such a tx was never created but as we don't know if it will happen before activation date we cannot
+                // enforce the bug fix which represents a rule change before the activation date.
+                handleIssuanceCandidateOutput(tempTxOutput);
+            } else if (isHardForkActivated(tempTxOutput) && isBlindVoteFeeOutput(tempTxOutput)) {
+                // After the hard fork activation we fix a bug with a transaction which would have interpreted the
+                // issuance output as BSQ if the availableInputValue was >= issuance amount.
+                // Such a tx was never created but as we don't know if it will happen before activation date we cannot
+                // enforce the bug fix which represents a rule change before the activation date.
```

Comment:
Comments are switched, vote fee comment for issuance and vice versa

Reply:
Will fix